### PR TITLE
Food item card

### LINKED ISSUE #7: Component > food item card
this is the food item card in the foods list, 
should receive props `title`, `image`, ...etc.

when the card clicked, it should go to route `/food/[id]`

no need to worry about the favorite button functionality for now, 

image

### PR DESCRIPTION
**Pull request by Adnan-961**
Feat: Added Food item card
Style: added separate style files for the component.
Comment: couldnt let the whole card to be clicked since it wasnt letting me use `useHistory()` because of the version. , only the food image is clickable. also the fav button can be clicked, but has no functionality yet.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+  margin-right: auto;
+  width: 40%;
+}
 .App {
   text-align: center;
 }
-
+.class{
+  background-color: red;
+}
```
Comment (osamaakb) [inline src/App.css]: remove these and use tailwind

File: src/components/Food-item-card/FoodItem.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import React from "react"; //17.0.2
+import  { useState } from "react";
+import PropTypes from "prop-types";
+import "./style.css";
+import {BrowserRouter as Router,Link} from "react-router-dom";
+function FoodItem(FoodItem) {
```
Comment (osamaakb) [inline src/components/Food-item-card/FoodItem.jsx]: please replace the props name with something more reasonable not the components name
```suggestion
function FoodItem(props) {
```

File: src/components/Food-item-card/style.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
+/* ul,ol,li{
+    list-style-type: disc !important;
+  } */
```
Comment (osamaakb) [inline src/components/Food-item-card/style.css]: please remove the comments

File: src/components/Food-item-card/style.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    margin-right: auto;
+    width: 100%;
+    height: 150px;
+  }
+
+
+  .icons{
+  
+    width: 15px !important;
+    height: 15px !important;
```
Comment (osamaakb) [inline src/components/Food-item-card/style.css]: why using css and you have tailwind, whyyyyyy?

File: src/components/category/Category.jsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+// Props used for testing
+// let obj = {
+//   Image:"https://image.flaticon.com/icons/png/128/3823/3823393.png",
+//   id:"5",
+//   Title:"Veggie"
+// };
```
Comment (osamaakb) [inline src/components/category/Category.jsx]: remove the comments

**Review by osamaakb — CHANGES_REQUESTED** [review]
Format and lint before staging

**Comment by osamaakb** [comment]
folder name should be all capitale no `-` included

**Comment by Adnan-961** [comment]
hey, @osamaakb , 
this pr is related to #23 
test1
test2
code used for testing :

 let obj = {
    img: "https://nomadparadise.com/wp-content/uploads/2020/08/lebanese-food-1-1024x683.jpg",
    title: "Tabbouleh",
    id: "3",
    time: "34",
    cal: "330",
  };
<Food {...obj} />

**Comment by osamaakb** [comment]
Looks good, few comments on the design, 
add more padding-top for the title and heart, 
the card bg should be white

**Comment by osamaakb** [comment]
the width should be like 50%

**Review by osamaakb — APPROVED** [review]
Good job @Adnan-961

**Pull request merged**